package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Universal front-end projects to complement addons-server.",
   "main": "index.js",
   "private": true,
+  "engines": {
+    "node": "^6.11.1"
```

Comment:
Can this just be `"node": "6"`?  Otherwise, I supposed we'd have to update it anytime we move to a new minor release of Node 6.